### PR TITLE
Client view all applications home

### PR DESCRIPTION
**Pull request by thomasalemayehu**

### CODE REVIEW

**Comment by thomasalemayehu** [comment]
I was thinking of adding a PULL TO REFRESH feature for user's side to update the admission status. Adding a whole watcher just to change the admission status seemed a bit of an over kill. WHAT DO YOU THINK?

File: client/lib/application/view_application/view_application_bloc.dart [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  ) async* {
+    final IApplicationRepository _iApplicationRepository =
+        getIt<IApplicationRepository>();
+    yield* event.map(
+      started: (e) async* {
+        // Start Loader animation
+        yield state.copyWith(isFetching: true);
+
+        // Mock Api call
+        await Future.delayed(const Duration(seconds: 2));
```
Comment (nati-m-afw) [inline client/lib/application/view_application/view_application_bloc.dart:35]: what is its purpose?

**Review by nati-m-afw — APPROVED** [review]

**Comment by nati-m-afw** [comment]
> I was thinking of adding a PULL TO REFRESH feature for user's side to update the admission status. Adding a whole watcher just to change the admission status seemed a bit of an over kill. WHAT DO YOU THINK?

Yeah I agree, since it's just a single event.

File: client/lib/application/view_application/view_application_bloc.dart (hunk 1 above)
Comment (thomasalemayehu) [reply client/lib/application/view_application/view_application_bloc.dart:35]: Umm Will remove it.. was just to check the loaders functionality

**Review by MichaelAlemayehu1 — APPROVED** [review]

**Pull request merged**